src/cmap/command_monitoring_events.ts

```diff
@@ -153,6 +184,9 @@ export class CommandFailedEvent {
     this.commandName = commandName;
     this.duration = calculateDurationInMs(started);
     this.failure = maybeRedact(commandName, cmd, error) as Error;
+    if (serverConnectionId) {
+      this.serverConnectionId = serverConnectionId;
```

Comment:
Same question about defaulting to null here and the rest of the event classes.

Reply:
No problem in defaulting to null here. It now defaults to null.